test/integration/kpatch-test

```diff
@@ -51,7 +51,7 @@ unset CCACHE_HASHDIR
 KPATCHBUILD="$ROOTDIR"/kpatch-build/kpatch-build
 ERROR=0
 LOG=test.log
-rm -f $LOG
+rm -f $(dirname ${LOG})/*$(basename ${LOG})
```

Comment:
Using dirname here seems a bit overkill, since the test log is always in the current directory.  If we ever decided to put the logs in another directory I think it would be simpler to just add a LOGDIR variable at that point.

Also maybe we can just use a .log convention so we don't need all the basenames everywhere.  Then this can just be `rm -f *.log` which is much easier to read.

Reply:
The convention in this file is that we don't use the `{` brackets, so in general e.g. use `$LOG` instead of `${LOG}`.